doc/iterations/2025.1/update_table_description_from_representation.adoc

```diff
@@ -0,0 +1,79 @@
+= (M) Shape
+
+== Problem
+
+On a table representation, we want to be able to modify the specification used as description of a table and then reload the table with this new specification.
+
+== Key Result
+
+Enable user to modify the description used by a table representation and update existing representation with the new description.
+
+== Solution 1 - Force update all the representation when creating a fork of the description
+
+From a table representation, an user will be able to click a setting button to be redirected to a new studio that contain the description of the current table. After updating this new studio, the user can go back to his previous representation and find it updated with the new description. Other users viewing the table will be redirected to the updated representation.
+
+=== Scenario
+
+==== An user wants to modify the description used by the table he's viewing
+
+* The user open a table representation
+* The user click on the setting button
+* He is redirected to a new studio that is a fork of the one used by the representation he had open
```

Comment:
containing a fork of the table description used to create the representation he had open